templates/registration/login.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Login</h1>
+
+  {% if form.errors %}
+    <p class="error">Oops! Something goes wrong!</p>

```

Comment:
A better way to say that authentification went wrong is: "Incorrect username or password" or something similar